app/controllers/phylo_trees_controller.rb

```diff
@@ -118,6 +118,7 @@ def show
     end
   end
 
+  # NOTE: this is used by the report page as well as phylo trees
```

Comment:
There is a Jira ticket to fix this soon: https://jira.czi.team/browse/IDSEQ-2127

Reply:
thanks, added a comment